build.fsx

```diff
@@ -141,26 +141,11 @@ Target "SourceLink" (fun _ ->
 // Build a NuGet package
 
 Target "NuGet" (fun _ ->
-    let fsiVersion = GetPackageVersion "packages" "FSharp.Compiler.Service"
-    let scriptcsVersion = GetPackageVersion "packages" "ScriptCs.Hosting"
-    NuGet (fun p -> 
+    Paket.Pack (fun p ->
         { p with   
-            Authors = authors
-            Project = project
-            Summary = summary
-            Description = description
```

Comment:
It might not be a bad idea to make passing via the command line an option - having said that, for solutions with lots of projects the template file or something similar is about the only sane option I've found so far!